dddwa/dddperth-website
# Adds She Codes community partner

Adds the She Codes logo and data so they appear as a community partner.

### Testing
* View http://localhost:3000/
* Scroll down to sponsorship area
* You'll see She Codes in Community Partners
